### PR TITLE
[Conv2d] Conv2d Padding Fix

### PR DESCRIPTION
**Pull request by zhoonit**
- [Conv2d] Conv2d Padding Fix

```
This patch resolves conv2d calculation is crooked in some cases

**Changes proposed in this PR:**
- Conv2d Fix
- Padding::Compute Fix

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Co-authored-by: Parichay Kapoor <pk.kapoor@samsung.com>
Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

### CODE REVIEW

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Review by kparichay — APPROVED** [review]
LGTM

File: nntrainer/layers/conv2d_layer.cpp [hunk 1]
```diff
@@ -81,8 +78,8 @@ static void col2im(const Tensor &col_matrix, const TensorDim &kdim,
   unsigned im_eff_width = im_width + pl + pr;
   image.setZero();
 
-  int h_stride_end = im_eff_height - eff_k_height - pb;
-  int w_stride_end = im_eff_width - eff_k_width - pr;
+  int h_stride_end = im_eff_height - eff_k_height - pt;
```
Comment (kparichay) [inline nntrainer/layers/conv2d_layer.cpp:81]: can you make this change for pooling2d layer as well.

Comment (zhoonit) [reply nntrainer/layers/conv2d_layer.cpp:81]: Right... Updated :)

**Review by lhs8928 — APPROVED** [review]
LGTM

**Pull request merged**